graylog2-server/src/main/java/org/graylog/metrics/prometheus/PrometheusExporter.java

```diff
@@ -113,4 +114,15 @@ private Collector createCollector(List<MapperConfig> mapperConfigs) {
                 new CustomMappingSampleBuilder(mapperConfigs)
         );
     }
+
+    public Collector createCollector() {
```

Comment:
If this is just for testing then you might annotate it as `@VisibleForTesting`

Reply:
I see that `createCollector` is going to be called in Forwarder, so I'll resolve this.